src/GafferCycles/IECoreCyclesPreview/Renderer.cpp

```diff
@@ -3128,11 +3120,18 @@ class CyclesRenderer final : public IECoreScenePreview::Renderer
 			InternedString cryptoMaterial;
 			bool hasShadowCatcher = false;
 			bool hasDenoise = false;
+			const bool useIEDisplay = std::any_of(
+				m_outputs.begin(), m_outputs.end(),
+				[] ( const auto &output ) { return output.second->m_useIEDisplay; }
+			);
 			for( auto &coutput : m_outputs )
 			{
-				if( ( m_renderType != Interactive && coutput.second->m_interactive ) ||
-					( m_renderType == Interactive && !coutput.second->m_interactive ) )
+				if( coutput.second->m_useIEDisplay != useIEDisplay )
```

Comment:
Yeah, I was in two minds on that.

<img src=https://imgs.xkcd.com/comics/workflow.png>

It's certainly plausible that someone has been relying on the side-effects of the bug, whether inadvertently or deliberately. So for me the question is whether the greatest good (and/or least damage) comes from fixing it earlier, or holding the fix till 1.5. And I don't think we have sufficient information to know.

If this was in Arnold, for me a think the balance would be swinging towards holding it for 1.5. That's because I'm responsible very directly to a couple of large Arnold-based production pipelines where unexpected changes in behaviour could be quite disruptive. I know much less about the likelihood of disruption in the Cycles pipelines out there. My general assumption is that Cycles users are probably more accustomed to bugs, and more nimble in side-stepping them, so the balance may swing the other way. But that's just a guess...

Happy to retarget to `main` if you think that's better. Or to consider opinions from anyone else with more skin in this particular game.